### PR TITLE
For section php value NULL skip current option.

### PR DESCRIPTION
**Pull request by h4kuna**

### CODE REVIEW

**Comment by h4kuna** [comment]
Here is thread from [forum](https://forum.nette.org/en/24067-neon-remove-php-settings-for-forward-compatibility)

File: src/DI/Extensions/PhpExtension.php [hunk 1]
```diff
@@ -20,7 +20,10 @@ public function afterCompile(Nette\PhpGenerator\ClassType $class)
 	{
 		$initialize = $class->getMethod('initialize');
 		foreach ($this->getConfig() as $name => $value) {
-			if (!is_scalar($value)) {
+			if($value === NULL) {
```
Comment (lookyman) [inline src/DI/Extensions/PhpExtension.php]: missing space after `if``

Comment (h4kuna) [inline src/DI/Extensions/PhpExtension.php]: Fixed. Thanks.

**Comment by dg** [comment]
Fix commit message please, i.e `PhpExtension: NULLs are skipped` or whatever.

**Comment by h4kuna** [comment]
Commit message is changed.

**Comment by dg** [comment]
Thanks

**Pull request merged**